packages/ui-extensions-react/src/surfaces/checkout.ts

```diff
@@ -57,7 +57,6 @@ export type {
   MaybeShorthandProperty,
   NonPresentationalAccessibilityRole,
   OverlayActivatorProps,
-  PaymentMethod,
```

Comment:
Remove change from release. This will be added in `2025-07-RC` right after the stable `2025-04`.